containers/sandbox/Dockerfile

```diff
@@ -19,3 +19,6 @@ RUN apt-get update && apt-get install -y \
     && rm -rf /var/lib/apt/lists/*
 
 RUN service ssh start
```

Comment:
Does this actually do anything? I imagine we need to start the ssh service when the container starts running, not at build time

Reply:
Ok.. Claude Opus said no:

> In a Dockerfile, the `RUN` instruction executes commands during the image build process. The command `service ssh start` is typically used to start the SSH service on a Linux system.
> 
> However, in the context of a Dockerfile, using `RUN service ssh start` is generally not recommended or necessary for the following reasons:
> 
> 1. The `RUN` instruction in a Dockerfile is executed during the image build process, not when a container is started from the built image. Any processes started with `RUN` will not be running when the container is actually launched.
> 
> 2. The `service` command is used to manage system services on a running Linux system. Inside a Dockerfile, there is no running system services as it is just a build environment.
> 
> 3. If you need to run the SSH service inside your container, you should use the `CMD` or `ENTRYPOINT` instruction to specify the command that starts the SSH service when the container is launched.
> 
> Instead of using `RUN service ssh start`, a better approach would be to use something like:
> 
> ```dockerfile
> # Install SSH server package
> RUN apt-get update && apt-get install -y openssh-server
> 
> # Configure and enable SSH
> RUN mkdir /var/run/sshd
> RUN echo 'root:password' | chpasswd
> RUN sed -i 's/#PermitRootLogin prohibit-password/PermitRootLogin yes/' /etc/ssh/sshd_config
> 
> # Expose SSH port
> EXPOSE 22
> 
> # Start SSH service when the container launches
> CMD ["/usr/sbin/sshd", "-D"]
> ```
> 
> In this example, the SSH server package is installed, configured, and set to start when the container is launched using the `CMD` instruction.
> 
> Keep in mind that running an SSH service inside a container is often unnecessary and can introduce security risks. It's generally recommended to use alternative methods such as Docker's built-in exec functionality or tools like Docker Compose for accessing and managing containers.

I will remove it!